start-wizard.sh

```
@@ -0,0 +1,240 @@
+    [ -n "$desc" ] && printf "  ${DIM}%s${NC}" "$desc"
+    printf "\n"
+    printf "  Keep [${GREEN}%s${NC}] or enter new value: " "$current"
+
+    local input
+    read -r input </dev/tty || input=""
+    local chosen="${input:-$current}"
+    printf '%s=%s\n' "$key" "$chosen" >> "$TMPFILE"
+
+    # Warn and force full if fast sync mode is set
+    if [ "$key" = "SYNC_MODE" ] && [ "$chosen" = "fast" ]; then
+        printf "\n  ${BOLD}${RED}WARNING:${NC} SYNC_MODE=fast is currently broken and not supported.\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)